import_export/resources.py

```diff
@@ -739,8 +739,11 @@ def widget_from_django_field(cls, f, default=widgets.Widget):
             result = functools.partial(widgets.ManyToManyWidget,
                                        model=f.rel.to)
         if internal_type in ('ForeignKey', 'OneToOneField', ):
+            field = 'pk'
+            if f.remote_field.field_name:
+                field = f.remote_field.field_name
             result = functools.partial(widgets.ForeignKeyWidget,
-                                       model=f.rel.to)
+                                       model=f.related_model, field=field)
         if internal_type in ('DecimalField', ):
```

Comment:
I followed the fix, but I got  "NameError: name 'field' is not defined". I encounter this problem when I am upgrading Django from 1.8.5 to 2.0. What should I change? Great thanks beforehand